.github/workflows/build.yml

```diff
@@ -5,6 +5,9 @@ on:
     branches:
       - "release/**"
 
+  # TODO: Revert before merge
+  pull_request:
```

Comment:
Out of interest, what does this do and why does it need to be reverted before merge?

Reply:
We usually don't run the full release build on every pull request since that would take too much time. I'm enabling this temporarily now to check that the builds work, and then we're disabling them again.

Reply:
I see, thank you for the explanation!